packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -23,35 +23,170 @@ import fs from "fs";
 
 import { duration } from "../../../tasit-sdk/dist/testHelpers/helpers";
 
-const { ONE, TEN, ONE_HUNDRED, ONE_THOUSAND } = constants;
+const { ONE, TEN, ONE_HUNDRED, ONE_THOUSAND, BILLION, WeiPerEther } = constants;
 
-const createMultipleParcels = async (
-  landContract,
-  parcels,
-  beneficiaryAddress,
-  contractOwnerWallet
-) => {
-  let xArray = [];
-  let yArray = [];
+let network = process.env.NETWORK;
 
-  parcels.forEach(parcel => {
-    xArray.push(parcel.x);
-    yArray.push(parcel.y);
+let EVENTS_TIMEOUT;
+
+const config = require(`../config/${network}.js`);
+
+// https://stats.goerli.net/
+if (network === "goerli") {
+  gasParams = {
+    gasLimit: 8e6,
+    gasPrice: 1e10,
+  };
+  EVENTS_TIMEOUT = 5 * 60 * 1000;
+} else if (network === "ropsten") {
+  EVENTS_TIMEOUT = 5 * 60 * 1000;
+} else if (network === "development") {
+  network = "local";
+  EVENTS_TIMEOUT = 1000;
+}
+const {
+  LANDProxy,
+  EstateRegistry,
+  Marketplace,
+  GnosisSafe: GnosisSafeInfo,
+  MANAToken,
+} = TasitContracts[network];
+const { address: MANA_ADDRESS } = MANAToken;
+const { address: LAND_PROXY_ADDRESS } = LANDProxy;
+const { address: ESTATE_ADDRESS } = EstateRegistry;
+const { address: MARKETPLACE_ADDRESS } = Marketplace;
+const { address: GNOSIS_SAFE_ADDRESS } = GnosisSafeInfo;
+
+ConfigLoader.setConfig(config);
+
+const [minterWallet, sellerWallet] = accounts;
+const { address: sellerAddress } = sellerWallet;
+
+const manaContract = new Mana(MANA_ADDRESS);
+const landContract = new Land(LAND_PROXY_ADDRESS);
+const estateContract = new Estate(ESTATE_ADDRESS);
+const marketplaceContract = new Decentraland(MARKETPLACE_ADDRESS);
+const gnosisSafeContract = new GnosisSafe(GNOSIS_SAFE_ADDRESS);
+
+const provider = ProviderFactory.getProvider();
+
+const cancelSellOrder = async (nftAddress, id) => {
+  const action = marketplaceContract.cancelOrder(
+    nftAddress,
+    `${id}`,
+    gasParams
+  );
+  await action.waitForNonceToUpdate();
```

Comment:
Does a try/catch belong in here our outside where the `cancelSellOrder` function is used? I guess it depends if we want to return `false` or something if it doesn't work.

Reply:
I'm using higher-level try/catch on that and other operation that a specific return isn't needed and event listener style on otherwise.

Reply:
Got it - thanks. Resolving this.